content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -299,95 +378,30 @@ those identities.
-together, which is also known as {{< gloss >}}multi-mesh{{< /gloss >}}.
-
-### Single mesh
-
-The simplest Istio deployment is a single mesh. Within a mesh, service names are
-unique. For example, only one service can have the name `mysvc` in the `foo`
-namespace. Additionally, workload instances share a common identity since
-service account names are unique within a namespace, just like service names.
-
-A single mesh can span [one or more clusters](#cluster-models) and
-[one or more networks](#network-models). Within a mesh,
-[namespaces](#namespace-tenancy) are used for [tenancy](#tenancy-models).
-
-### Multiple meshes
-
-Multiple mesh deployments result from {{< gloss >}}mesh federation{{< /gloss >}}.
-
-Multiple meshes afford the following capabilities beyond that of a single mesh:
-
-- Organizational boundaries: lines of business
-- Service name or namespace reuse: multiple distinct uses of the `default`
-  namespace
-- Stronger isolation: isolating test workloads from production workloads
-
-You can enable inter-mesh communication with {{< gloss >}}mesh federation{{<
-/gloss >}}. When federating, each mesh can expose a set of services and
-identities, which all participating meshes can recognize.
-
-{{< image width="50%"
-    link="multi-mesh.svg"
-    alt="Multiple service meshes"
-    title="Multi-mesh"
-    caption="Multiple service meshes"
-    >}}
-
-To avoid service naming collisions, you can give each mesh a globally unique
-**mesh ID**, to ensure that the fully qualified domain
-name (FQDN) for each service is distinct.
-
-When federating two meshes that do not share the same
-{{< gloss >}}trust domain{{< /gloss >}}, you must
-{{< gloss "mesh federation">}}federate{{< /gloss >}}
-{{< gloss >}}identity{{< /gloss >}} and **trust bundles** between them. See the
-section on [Multiple Trust Domains](#trust-between-meshes) for an overview.
-
-## Tenancy models
-
-In Istio, a **tenant** is a group of users that share
-common access and privileges to a set of deployed workloads. Generally, you
-isolate the workload instances from multiple tenants from each other through
-network configuration and policies.
-
-You can configure tenancy models to satisfy the following organizational
-requirements for isolation:
+Istio does not have a formally defined notion of tenancy. However, you can achieve a great
+level of isolation across different teams through the use of different namespaces or even separate clusters.
+Such soft tenancy models can be used to satisfy the following organizational requirements for isolation:
 
 - Security
 - Policy
 - Capacity
 - Cost
 - Performance
 
-Istio supports two types of tenancy models:
+Istio supports two types of soft-tenancy models:
 
-- [Namespace tenancy](#namespace-tenancy)
-- [Cluster tenancy](#cluster-tenancy)
+- [Namespace isolation](#namespace-isolation)
+- [Cluster isolation](#cluster-isolation)
 
-### Namespace tenancy
+### Namespace isolation
```

Comment:
Why is `isolation` preferred to just `tenancy`?

Reply:
Because we dont have a formal tenancy concept. No one knows what a tenant is. Docs have been flying around about this "tenancy" thing. Isolation != tenancy. We shouldn't oversell this bad :)

Reply:
Still I think I'd rather keep the tenancy docs as-is until we finalize the new conceptual model for tenancy. I'm not sure this rename is an improvement.